src/components/ItemList/ItemList.module.css

```diff
@@ -10,3 +10,7 @@
 li {
   list-style-type: none;
 }
+
+.button__delete {
+  display: none;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5703286)